aggregator/cmd/main.go

```diff
@@ -69,6 +71,24 @@ func aggregatorMain(ctx *cli.Context) error {
 		return err
 	}
 
+	registry := agg.GetRegistry()
+	rpcServer := rpcserver.NewRpcServer(config.AggregatorServerIpPortAddr, agg, logger)
+	if registry != nil {
+		if err = rpcServer.EnableMetrics(registry); err != nil {
+			return err
+		}
+	}
+
+	go rpcServer.Start()
+
+	restServer := restserver.NewRestServer(config.AggregatorRestServerIpPortAddr, agg, logger)
+	if registry != nil {
+		if err = restServer.EnableMetrics(registry); err != nil {
+			return err
+		}
+	}
```

Comment:
Nit-pick: extra whitespace to match with the other one

Reply:
Done